server/routes/induction/update/previousWorkExperienceTypesUpdateController.ts

```diff
@@ -58,14 +57,20 @@ export default class PreviousWorkExperienceTypesUpdateController extends Previou
         logger.debug(
           'Previous Work Experiences changes are only removed types so Induction can be updated without asking for work experience details',
         )
-        // call the service to update the Induction
+
+        // create an updated InductionDto with any changes to Previous Work Experiences and then map it to a CreateOrUpdateInductionDTO to call the API
```

Comment:
Change the way the Previous Work Experience **Types** controller works re: submit handler....

Here, we are still only updating the Induction when there are only removals, but we are now doing it in the exact same way we do with all our other Update controllers; in that we map to the Induction (I've change the mapper method, see below), then map it to the CreateOrUpdateDto, then call the service.

Previously the updating of the Induction via the service call was in a private method because I'd envisaged calling it from different places for different circumstances (ie. for removals only, and for additions, and for changes to Other etc). This is now not the case, and I think the only time this submit hander will actually call the API to update the Induction is here when the only change is removals 👍

Reply:
So if they remove and add some experience types, then the changes won't be persisted until they've provided details for the first work experience on the subsequent screen?

Reply:
> So if they remove and add some experience types, then the changes won't be persisted until they've provided details for the first work experience on the subsequent screen?

Not sure yet ..... at the moment I'll tentatively say Yes .... but I'm not sure it works in all cases, and I want to bounce it with you after lunch.

It doesnt change/impact the changes I'm proposing in this PR though. I think we need these changes regardless